transcom/mymove
# B-18990 Add shipment locator officer pages KICKBACK

## [B-18990](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A909786)

## Summary
This PR adds shipment locator to all shipment cards from all officer views.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create any move or use any existing move
2. Login as any officer(TXO, SC)
3. Confirm the shipment locator is under the Shipment type. (Similar to the moveLocator for the customer)

### Frontend


## Screenshots
**_TOO viewing approved shipment with a diversion request:_**
<img width="1046" alt="Screenshot 2024-04-17 at 12 48 23 PM" src="https://github.com/transcom/mymove/assets/146969726/beeed871-5a37-46e1-a5e6-405e229633bb">


**_TOO viewing approved move under Move task order tab:_**
<img width="1066" alt="Screenshot 2024-04-17 at 12 48 13 PM" src="https://github.com/transcom/mymove/assets/146969726/7a451d8b-0a4d-485f-9887-ca5c0eb8e432">


**_SC viewing move details tab of not submitted move:_**
<img width="1012" alt="Screenshot 2024-04-17 at 1 19 47 PM" src="https://github.com/transcom/mymove/assets/146969726/d600913f-6f66-4570-a253-030306a1ecec">
